supervisor/api/const.py

```diff
@@ -60,3 +65,10 @@
 ATTR_USE_NTP = "use_ntp"
 ATTR_USERS = "users"
 ATTR_VENDOR = "vendor"
+
+
+class BootName(StrEnum):
```

Comment:
I think I'd go for BootSlot instead but either way works for me

```suggestion
class BootSlot(StrEnum):
```

Reply:
@agners I was trying to stick with the names I was seeing in Rauc I guess. Rauc seemed to be ID'ing boot slots with names like `kernel.0` and `kernel.1` and referred to `A` and `B` as boot names in the status.

Reply:
Technical it's called boot slot over all implementation. Embedded Developers vs. Application Developers - I guess on Rauc, there where more application developers.

Reply:
Reading documentation such as https://rauc.readthedocs.io/en/latest/using.html#manually-switch-to-a-different-slot seems to me that RAUC is using slot for the whole active boot "configuration" as well. Also chapter 2 is mostly talking about slots https://rauc.readthedocs.io/en/latest/basic.html 🤔